packages/ui-concerto/src/components/fields.js

```diff
@@ -65,27 +72,45 @@ export const ConcertoInput = ({
       console.warn(validationError.message);
     }
   }
-  return <Form.Field key={`field-${id}`} required={required} error={error}>
-    <ConcertoLabel key={`label-${id}`} skip={skipLabel} name={applyDecoratorTitle(field)} htmlFor={id} />
-    <Input
-      id={id}
-      type={type}
-      readOnly={readOnly}
-      value={value}
-      onChange={(e, data) => onFieldValueChange(
-        { ...data, value: parseValue(data.value, field.getType()) },
-        id
-      )
-      }
-      key={`input-${id}`}
-    />
-  </Form.Field>;
+  return (
+    <Form.Field key={`field-${id}`} required={required} error={error}>
+      <ConcertoLabel
+        key={`label-${id}`}
+        skip={skipLabel}
+        name={applyDecoratorTitle(field)}
+        htmlFor={id}
+      />
+      {!textOnly ? (
+        <Input
+          id={id}
+          type={type}
+          readOnly={readOnly}
+          value={value}
+          onChange={(e, data) =>
+            onFieldValueChange(
+              { ...data, value: parseValue(data.value, field.getType()) },
+              id
+            )
+          }
+          key={`input-${id}`}
+        />
+      ) : (
+        <div className="textOnly">
+          <label value={value} key={`input-${id}`}>
+            {" "}
```

Comment:
What's the purpose of this, please?

Reply:
I will remove the package-lock.json file. And yes it appears my code editor made some unintentional changes. I apologise for that. Shall I undo them? 

The changes in the fields.js file ensure that when the textOnly mode is enabled, the UI shows only a label instead of a disabled input field.

Reply:
I have removed the package-lock.json file. Do let me know if any other changes are required. Thanks

Reply:
The DCO sign-off check is not completed even though I signed off while committing the changes. Could you please guide me on how to proceed as this is the first time I have contributed to an open-source project and I am a little confused? @mttrbrts

Reply:
Apologies for the delay in replying over the holidays.

Thanks for your changes.

Re the DCO sign-off if you follow the `Details` link to the failed check, GitHub gives you some hints to resolve the issue. Let me know if you continue to have issues.
![image](https://user-images.githubusercontent.com/7544022/148196614-ba27c77d-0fe5-4469-86ac-d0705da09445.png)
https://github.com/accordproject/web-components/runs/4642124418

Reply:
Also, I should be more specific regarding `package-lock.json`. We still need the file, but just not your changes to it. In your latest commits the file has been deleted entirely.

Reply:
The problem is that I am unable to access commit [(https://github.com/accordproject/web-components/pull/362/commits/55193d39ad15d8377538cc530e0dceca57b4759c)] which is responsible for these issues in my local branch. That is the reason why I cannot sign-off or revert it. I am extremely sorry for the inconvenience caused. May I close this PR and open up a new one?

Reply:
Yes, of course, whatever is easiest for you. You should be able to close your own PRs.